### ISSUE TITLE
Path Constraint support

### ISSUE THREAD

**Issue by Nestach** [pixijs/pixi-spine#173]
Hi!

I was testing out Path Constraints to have a mesh with bones to follow along a path. But running it in Pixi didn't really work. It got stuck on the first frame but played the length of the animation. So my question is: are Path Constraints supported in Pixi?

**Comment by ivanpopelyshev** [comment]
Which version of spine editor do you use?

**Comment by Nestach** [comment]
3.5.51

**Comment by ivanpopelyshev** [comment]
Ok, then something is wrong , definitely

**Comment by Nestach** [comment]
Indeed, so if I have the latest version of Pixi/pixi-spine then Path Constraints should work right?

**Comment by ivanpopelyshev** [comment]
It depends only on pixi-spine and spine editor version. 3.5 has to work here!

**Comment by Nestach** [comment]
I will check the version of my pixi-spine, I suspect it might be the issue. I'll do some updates and report back

**Comment by valtver** [comment]
Hi,
@Nestach Can you check the path setting "Constant speed"? Try to leave it checked.
@ivanpopelyshev Faced the same issue with paths. Non-constant speed path is not interpolated in runtime.

**Comment by ivanpopelyshev** [comment]
@valtver good idea. i will check in my evening (in 8 hours or so)

**Comment by Nestach** [comment]
Ah I will have a look

**Comment by ivanpopelyshev** [comment]
@valtver

Please check it in latest version ! (in "bin" folder). if its ok, ill upload it to npm

**Comment by valtver** [comment]
@ivanpopelyshev 

Checked! Works fine now...everything moves :)

Спасибо :)

**Comment by ivanpopelyshev** [comment]
@valtver so now i can just make v3.6 features on top of it, отлично.
